tasks/kernel_matrix_testing/compiler.py

```diff
@@ -89,8 +89,12 @@ def exec(self, cmd: str, user="compiler", verbose=True, run_dir: PathOrStr | Non
             cmd = f"cd {run_dir} && {cmd}"
 
         self.ensure_running()
-        return self.ctx.run(
-            f"docker exec -u {user} -i {self.name} bash -c \"{cmd}\"", hide=(not verbose), warn=allow_fail
+
+        # Set FORCE_COLOR=1 so that termcolor works in the container
```

Comment:
Kind of related to this. I noticed that if I ran `inv kmt...` without `-e`, I still saw the commands from things running in the container. Is there a way we can detect the `echo` setting and propagate it correctly?

Reply:
Fixed!